queries/rust/injections.scm

```diff
@@ -71,7 +73,8 @@
     (#eq? @_new "new"))
   arguments: (arguments
     (array_expression
-      (raw_string_literal) @injection.content))
+      (raw_string_literal
+        (string_content)) @injection.content))
```

Comment:
This is still being applied to `raw_string_literal` and not string_content

Reply:
Oops🤦‍♂️thanks